pkg/envoy/cell.go

```diff
@@ -76,6 +77,7 @@ func (r envoyProxyConfig) Flags(flags *pflag.FlagSet) {
 	flags.Bool("disable-envoy-version-check", false, "Do not perform Envoy version check")
 	flags.Int("proxy-prometheus-port", 0, "Port to serve Envoy metrics on. Default 0 (disabled).")
 	flags.Int("proxy-admin-port", 0, "Port to serve Envoy admin interface on.")
+	flags.Uint("envoy-access-log-buffer-size", 4096, "Envoy access log buffer size in bytes. Default 4096B.")
```

Comment:
There's no need to document the default in the description. The default value (if different than the types zero value) is added automatically when generating the docs.

e.g. see diff in `Documentation/cmdref/cilium-agent.md` - currently looks like this (with duplicate)

```
[...] Envoy access log buffer size in bytes. Default 4096B. (default 4096)
```

Reply:
Makes sense; just force-pushed https://github.com/cilium/cilium/commit/f35b38a5b691c6cb58ca0a03c7ac31992071d8af which fixes this